bootstrap-starter-data/_components.yml

```diff
@@ -5,9 +5,9 @@ _components:
       new:
         title: ''
       new-author:
-        title: ''
+        title: 'testing2'
```

Comment:
Note: every time someone creates a new page that uses this component, it will have testing2 as default value. Thinking in a production env, do we want this?

Reply:
You are right, in the new commit that I uploaded these recommendations are fixed, Thank you :)!